Dockerfile

```diff
@@ -1,25 +1,41 @@
-FROM node:8.2.1-alpine
+FROM node:8.9
 
 # install yarn
 ENV PATH /root/.yarn/bin:$PATH
 
-RUN apk update \
-  && apk add curl bash binutils tar git python3 \
-  && rm -rf /var/cache/apk/* \
-  && /bin/bash \
-  && touch ~/.bashrc \
-  && curl -o- -L https://yarnpkg.com/install.sh | bash
+RUN apt-get update \
+  && apt-get install -y curl bash binutils tar git python3 locales \
+  && rm -r /var/lib/apt/lists/*
+
+RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
+  dpkg-reconfigure --frontend=noninteractive locales && \
+  update-locale LANG=en_US.UTF-8
+
+ENV LANG en_US.UTF-8
+
+RUN curl -o- -L https://yarnpkg.com/install.sh | bash
+
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python2.7 1 \
```

Comment:
Neither of these Python versions are supported by Home Assistant 🤔

Reply:
I checked the old docker, and noticed this:
```
Last login: Wed Feb 27 23:21:35 on ttys000
astaniscia@spa-079-l:~$ docker run -it node:8.2.1-alpine sh
/ # apk update && apk add python3
fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/community/x86_64/APKINDEX.tar.gz
v3.6.2-271-g168bada463 [http://dl-cdn.alpinelinux.org/alpine/v3.6/main]
v3.6.2-266-g0f1867409e [http://dl-cdn.alpinelinux.org/alpine/v3.6/community]
OK: 8443 distinct packages available
(1/11) Installing libbz2 (1.0.6-r5)
(2/11) Installing expat (2.2.0-r1)
(3/11) Installing libffi (3.2.1-r3)
(4/11) Installing gdbm (1.12-r0)
(5/11) Installing xz-libs (5.2.3-r0)
(6/11) Installing ncurses-terminfo-base (6.0_p20171125-r0)
(7/11) Installing ncurses-terminfo (6.0_p20171125-r0)
(8/11) Installing ncurses-libs (6.0_p20171125-r0)
(9/11) Installing readline (6.3.008-r5)
(10/11) Installing sqlite-libs (3.20.1-r1)
(11/11) Installing python3 (3.6.1-r3)
Executing busybox-1.26.2-r5.trigger
OK: 71 MiB in 24 packages
/ # python3 --version
Python 3.6.1
/ #
```

can you confirm  this version? **Python 3.6.1**